functions.php

```diff
@@ -2,18 +2,22 @@
 
 
 if ( ! class_exists( 'Timber' ) ) {
-	add_action( 'admin_notices', function() {
-		echo '<div class="error"><p>Timber not activated. Make sure you activate the plugin in <a href="' . esc_url( admin_url( 'plugins.php#timber' ) ) . '">' . esc_url( admin_url( 'plugins.php') ) . '</a></p></div>';
-	});
+	add_action(
+		'admin_notices', function() {
+			printf('<div class="error"><p>Timber not activated. Make sure you activate the plugin in <a href="' . esc_url( admin_url( 'plugins.php#timber' ) ) . '">' . esc_url( admin_url( 'plugins.php' ) ) . '</a></p></div>');
```

Comment:
`s/printf()` allows you to use placeholders, so HTML structure is easier to see as it's not concatenated. Also, it's better to use text as link instead of displaying scary admin URL.

So:

```
printf( '<div class="error"><p>Timber not activated. Make sure you activate the plugin in <a href="%s">Plugins menu</a></p></div>',
	esc_url( admin_url( 'plugins.php#timber' ) )
);
```

Reply:
I have added above